sNode/main.c

```diff
@@ -0,0 +1,13 @@
+#include "sNode.c"
+
+int main(void)
+{
+	sNode *node = NULL;
+	CreateNode(&node, "z");
```

Comment:
I know it's just a POC, but generally, passing a `const char*` (in this case string literals) to functions taking a `char *` is problematic